spec/core_functions/color/same.hrx

```diff
@@ -0,0 +1,28 @@
+<===> known_space/input.scss
+a {b: same(hsl(0 100% 25.1%), 'red')}
```

Comment:
```suggestion
a {b: same(hsl(0 100% 25.1%), red)}
```
Colors should be colors, not strings. You can remove the quotes here and throughout.